init.ps1

```diff
@@ -1,57 +0,0 @@
-<#
```

Comment:
Since this file has been deleted does it mean that we aren't required to run `./init.cmd` for dependency changes?